detox/local-cli/__mocks__/executable

```diff
@@ -4,22 +4,32 @@ const fs = require('fs');
 
 const DETOX_REGEXP = /^DETOX_(?!DISABLE_POD_INSTALL|DISABLE_POSTINSTALL)/;
 
-const env = Object.keys(process.env)
-  .filter(k => DETOX_REGEXP.test(k.toUpperCase()))
-  .reduce((acc, key) => {
-    acc[key] = process.env[key];
-    return acc;
-  }, {});
+function main() {
```

Comment:
Modernized this mock executable so it can go to "sleep" – that helps to test a few extra cases.